src/main/java/graphql/TypeResolutionEnvironment.java

```diff
@@ -24,24 +27,20 @@
     private final GraphQLSchema schema;
     private final Object context;
     private final GraphQLContext graphQLContext;
+    private final Object localContext;
     private final DataFetchingFieldSelectionSet fieldSelectionSet;
 
-    public TypeResolutionEnvironment(Object object,
```

Comment:
Yes this is breaking API - but this should not have been made public - a builder pattern (which we started AFTER this class was invented) would have not made this a breaking change.

We really dont want consumers making these

Reply:
we could got the whole hog and merge the two classes and put the builder int this class.  I chose not to do this - progress over perfection